src/utils.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module horizontal-rule/utils
```

Comment:
Such files don't make sense until we see that we need to make such functions reusable. So far, they are not reused and I don't see an immediate need to reuse them. Hence, KISS.

I know that in some other packages such functions are exposed, but there's either a clear reason for that or someone was overly eager to expose them which was a mistake.

PS. This file would have to be covered with tests and now it's not. See? :P No point in exposing such utils. It only forces us to code more.

Reply:
Thanks for the clarification.